.gitignore

```diff
@@ -19,3 +19,4 @@ glad.zip
 /Box_test_diff.ppm
 /ghostty.qcow2
 /build.zig.zon2json-lock
+/blob
```

Comment:
Any reason we can't just dump this into the pwd and ignore the filename? "Blob" seems a bit random.